kratos/tests/cpp_tests/strategies/schemes/test_residual_based_adjoint_bossak_scheme.cpp

```diff
@@ -510,6 +787,53 @@ class ResponseFunction : public AdjointResponseFunction
             rSensitivityGradient(0) = 0.;
         }
 
+        void CalculateGradient(const Condition& rAdjointCondition,
+                               const Matrix& rResidualGradient,
+                               Vector& rResponseGradient,
+                               const ProcessInfo& rProcessInfo) override
+        {
+            rResponseGradient.resize(1, false);
+            rResponseGradient(0) = 0.0;
```

Comment:
maybe you can comment here that the full contribution is assembled by the element

Reply:
done :)